src/main/resources/logback.xml

```diff
@@ -0,0 +1,52 @@
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="AllAppender" class="ch.qos.logback.core.FileAppender">
+        <file>log/all.log</file>
+        <append>false</append>
+        <immediateFlush>true</immediateFlush>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="ModelAppender" class="ch.qos.logback.core.FileAppender">
+        <file>log/model.log</file>
+        <append>true</append>
+        <immediateFlush>true</immediateFlush>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="EvaluationAppender" class="ch.qos.logback.core.FileAppender">
+        <file>log/evaluator.log</file>
+        <append>true</append>
+        <immediateFlush>true</immediateFlush>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="INFO">
+        <appender-ref ref="STDOUT"/>
+        <appender-ref ref="AllAppender"/>
+    </root>
+
+    <logger name="org" level="INFO"/>
```

Comment:
why u need to set this if root is info ?

Reply:
Good question 👍 
org is using its own logger with debug or trace as its level but it doesn't use any appender, it just propagates the log that it generates to the root.

What I mean is that the logger of the org package will accept any log with a level debug and it will pass it to the root which will append it using its appender even if the level of the root is info.